draft-irtf-cfrg-vdaf.md

```diff
@@ -4833,12 +4831,12 @@ tightly as possible. The encoded public share is structured as follows:
 
 ~~~ tls-presentation
 struct {
-    Poplar1Seed seed;
+    opaque seed[16];
     Poplar1FieldInner payload[Fi * Poplar1.Idpf.VALUE_LEN];
 } Poplar1CWSeedAndPayloadInner;
 
 struct {
-    Poplar1Seed seed;
+    opaque seed[32];
```

Comment:
I don't think this is right, because `extend()` always returns a seed of `KEY_SIZE` bytes, and this is the XOR of two such seeds.

Reply:
Yup :/ fixed.